Now using node v20.5.0 (npm v9.8.0)
Running pre-execution tasks...
node:internal/modules/cjs/loader:1051
  throw err;
  ^

Error: Cannot find module '/root/server.js'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1048:15)
    at Module._load (node:internal/modules/cjs/loader:901:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:83:12)
    at node:internal/main/run_main_module:23:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v20.5.0
Now using node v14.20.0 (npm v6.14.17)
Now using node v20.5.0 (npm v9.8.0)
Running pre-execution tasks...
node:internal/modules/cjs/loader:1051
  throw err;
  ^

Error: Cannot find module '/root/server.js'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1048:15)
    at Module._load (node:internal/modules/cjs/loader:901:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:83:12)
    at node:internal/main/run_main_module:23:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v20.5.0
Now using node v14.20.0 (npm v6.14.17)
Now using node v20.5.0 (npm v9.8.0)
Running pre-execution tasks...
node:internal/modules/cjs/loader:1051
  throw err;
  ^

Error: Cannot find module '/root/server.js'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1048:15)
    at Module._load (node:internal/modules/cjs/loader:901:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:83:12)
    at node:internal/main/run_main_module:23:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v20.5.0
Now using node v14.20.0 (npm v6.14.17)
Now using node v20.5.0 (npm v9.8.0)
Running pre-execution tasks...
Now using node v20.5.0 (npm v9.8.0)
Running pre-execution tasks...
fatal: not a git repository (or any of the parent directories): .git
node:internal/errors:865
  const err = new Error(message);
              ^

Error: Command failed: git add .
fatal: not a git repository (or any of the parent directories): .git

    at checkExecSyncError (node:child_process:887:11)
    at execSync (node:child_process:959:15)
    at updateGitRepo (/root/proxy/server.js:86:9) {
  status: 128,
  signal: null,
  output: [
    null,
    Buffer(0) [Uint8Array] [],
    Buffer(69) [Uint8Array] [
      102,  97, 116,  97, 108,  58,  32, 110, 111, 116,  32,
       97,  32, 103, 105, 116,  32, 114, 101, 112, 111, 115,
      105, 116, 111, 114, 121,  32,  40, 111, 114,  32,  97,
      110, 121,  32, 111, 102,  32, 116, 104, 101,  32, 112,
       97, 114, 101, 110, 116,  32, 100, 105, 114, 101,  99,
      116, 111, 114, 105, 101, 115,  41,  58,  32,  46, 103,
      105, 116,  10
    ]
  ],
  pid: 372605,
  stdout: Buffer(0) [Uint8Array] [],
  stderr: Buffer(69) [Uint8Array] [
    102,  97, 116,  97, 108,  58,  32, 110, 111, 116,  32,
     97,  32, 103, 105, 116,  32, 114, 101, 112, 111, 115,
    105, 116, 111, 114, 121,  32,  40, 111, 114,  32,  97,
    110, 121,  32, 111, 102,  32, 116, 104, 101,  32, 112,
     97, 114, 101, 110, 116,  32, 100, 105, 114, 101,  99,
    116, 111, 114, 105, 101, 115,  41,  58,  32,  46, 103,
    105, 116,  10
  ]
}

Node.js v20.5.0
Now using node v14.20.0 (npm v6.14.17)
Now using node v20.5.0 (npm v9.8.0)
Running pre-execution tasks...
fatal: not a git repository (or any of the parent directories): .git
node:internal/errors:865
  const err = new Error(message);
              ^

Error: Command failed: git add .
fatal: not a git repository (or any of the parent directories): .git

    at checkExecSyncError (node:child_process:887:11)
    at execSync (node:child_process:959:15)
    at updateGitRepo (/root/proxy/server.js:86:9) {
  status: 128,
  signal: null,
  output: [
    null,
    Buffer(0) [Uint8Array] [],
    Buffer(69) [Uint8Array] [
      102,  97, 116,  97, 108,  58,  32, 110, 111, 116,  32,
       97,  32, 103, 105, 116,  32, 114, 101, 112, 111, 115,
      105, 116, 111, 114, 121,  32,  40, 111, 114,  32,  97,
      110, 121,  32, 111, 102,  32, 116, 104, 101,  32, 112,
       97, 114, 101, 110, 116,  32, 100, 105, 114, 101,  99,
      116, 111, 114, 105, 101, 115,  41,  58,  32,  46, 103,
      105, 116,  10
    ]
  ],
  pid: 374578,
  stdout: Buffer(0) [Uint8Array] [],
  stderr: Buffer(69) [Uint8Array] [
    102,  97, 116,  97, 108,  58,  32, 110, 111, 116,  32,
     97,  32, 103, 105, 116,  32, 114, 101, 112, 111, 115,
    105, 116, 111, 114, 121,  32,  40, 111, 114,  32,  97,
    110, 121,  32, 111, 102,  32, 116, 104, 101,  32, 112,
     97, 114, 101, 110, 116,  32, 100, 105, 114, 101,  99,
    116, 111, 114, 105, 101, 115,  41,  58,  32,  46, 103,
    105, 116,  10
  ]
}

Node.js v20.5.0
Now using node v14.20.0 (npm v6.14.17)
Now using node v20.5.0 (npm v9.8.0)
Running pre-execution tasks...
fatal: not a git repository (or any of the parent directories): .git
node:internal/errors:865
  const err = new Error(message);
              ^

Error: Command failed: git add .
fatal: not a git repository (or any of the parent directories): .git

    at checkExecSyncError (node:child_process:887:11)
    at execSync (node:child_process:959:15)
    at updateGitRepo (/root/proxy/server.js:86:9) {
  status: 128,
  signal: null,
  output: [
    null,
    Buffer(0) [Uint8Array] [],
    Buffer(69) [Uint8Array] [
      102,  97, 116,  97, 108,  58,  32, 110, 111, 116,  32,
       97,  32, 103, 105, 116,  32, 114, 101, 112, 111, 115,
      105, 116, 111, 114, 121,  32,  40, 111, 114,  32,  97,
      110, 121,  32, 111, 102,  32, 116, 104, 101,  32, 112,
       97, 114, 101, 110, 116,  32, 100, 105, 114, 101,  99,
      116, 111, 114, 105, 101, 115,  41,  58,  32,  46, 103,
      105, 116,  10
    ]
  ],
  pid: 376418,
  stdout: Buffer(0) [Uint8Array] [],
  stderr: Buffer(69) [Uint8Array] [
    102,  97, 116,  97, 108,  58,  32, 110, 111, 116,  32,
     97,  32, 103, 105, 116,  32, 114, 101, 112, 111, 115,
    105, 116, 111, 114, 121,  32,  40, 111, 114,  32,  97,
    110, 121,  32, 111, 102,  32, 116, 104, 101,  32, 112,
     97, 114, 101, 110, 116,  32, 100, 105, 114, 101,  99,
    116, 111, 114, 105, 101, 115,  41,  58,  32,  46, 103,
    105, 116,  10
  ]
}

Node.js v20.5.0
Now using node v14.20.0 (npm v6.14.17)
fatal: not a git repository (or any of the parent directories): .git
node:internal/errors:865
  const err = new Error(message);
              ^

Error: Command failed: git add .
fatal: not a git repository (or any of the parent directories): .git

    at checkExecSyncError (node:child_process:887:11)
    at execSync (node:child_process:959:15)
    at updateGitRepo (/root/proxy/server.js:86:9) {
  status: 128,
  signal: null,
  output: [
    null,
    Buffer(0) [Uint8Array] [],
    Buffer(69) [Uint8Array] [
      102,  97, 116,  97, 108,  58,  32, 110, 111, 116,  32,
       97,  32, 103, 105, 116,  32, 114, 101, 112, 111, 115,
      105, 116, 111, 114, 121,  32,  40, 111, 114,  32,  97,
      110, 121,  32, 111, 102,  32, 116, 104, 101,  32, 112,
       97, 114, 101, 110, 116,  32, 100, 105, 114, 101,  99,
      116, 111, 114, 105, 101, 115,  41,  58,  32,  46, 103,
      105, 116,  10
    ]
  ],
  pid: 377372,
  stdout: Buffer(0) [Uint8Array] [],
  stderr: Buffer(69) [Uint8Array] [
    102,  97, 116,  97, 108,  58,  32, 110, 111, 116,  32,
     97,  32, 103, 105, 116,  32, 114, 101, 112, 111, 115,
    105, 116, 111, 114, 121,  32,  40, 111, 114,  32,  97,
    110, 121,  32, 111, 102,  32, 116, 104, 101,  32, 112,
     97, 114, 101, 110, 116,  32, 100, 105, 114, 101,  99,
    116, 111, 114, 105, 101, 115,  41,  58,  32,  46, 103,
    105, 116,  10
  ]
}

Node.js v20.5.0
Now using node v14.20.0 (npm v6.14.17)
Now using node v20.5.0 (npm v9.8.0)
Running pre-execution tasks...
fatal: not a git repository (or any of the parent directories): .git
node:internal/errors:865
  const err = new Error(message);
              ^

Error: Command failed: git add .
fatal: not a git repository (or any of the parent directories): .git

    at checkExecSyncError (node:child_process:887:11)
    at execSync (node:child_process:959:15)
    at updateGitRepo (/root/proxy/server.js:86:9) {
  status: 128,
  signal: null,
  output: [
    null,
    Buffer(0) [Uint8Array] [],
    Buffer(69) [Uint8Array] [
      102,  97, 116,  97, 108,  58,  32, 110, 111, 116,  32,
       97,  32, 103, 105, 116,  32, 114, 101, 112, 111, 115,
      105, 116, 111, 114, 121,  32,  40, 111, 114,  32,  97,
      110, 121,  32, 111, 102,  32, 116, 104, 101,  32, 112,
       97, 114, 101, 110, 116,  32, 100, 105, 114, 101,  99,
      116, 111, 114, 105, 101, 115,  41,  58,  32,  46, 103,
      105, 116,  10
    ]
  ],
  pid: 384539,
  stdout: Buffer(0) [Uint8Array] [],
  stderr: Buffer(69) [Uint8Array] [
    102,  97, 116,  97, 108,  58,  32, 110, 111, 116,  32,
     97,  32, 103, 105, 116,  32, 114, 101, 112, 111, 115,
    105, 116, 111, 114, 121,  32,  40, 111, 114,  32,  97,
    110, 121,  32, 111, 102,  32, 116, 104, 101,  32, 112,
     97, 114, 101, 110, 116,  32, 100, 105, 114, 101,  99,
    116, 111, 114, 105, 101, 115,  41,  58,  32,  46, 103,
    105, 116,  10
  ]
}

Node.js v20.5.0
Now using node v14.20.0 (npm v6.14.17)
